src/main.cpp

```diff
@@ -15,13 +15,15 @@
 #include "inverter.hpp"
 #include "accumulator.hpp"
 #include "state_machine.hpp"
-#include "FlexCAN_util.hpp"
+#include "device_status.h"
+
+#define NEBUG
 
 // Metro timers for inverter:
 Metro timer_mc_kick_timer = Metro(50, 1);
 Metro timer_inverter_enable = Metro(2000, 1); // Timeout failed inverter enable
 Metro timer_motor_controller_send = Metro(100, 1);
-Metro timer_current_limit_send = Metro(500,1);
+Metro timer_current_limit_send = Metro(500, 1);
```

Comment:
probably need to increase this to be more responsive when driving on track
also need a disable flag for the current limting

Reply:
agreed